packages/samples/vuejswithcli/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "vuejswithcli",
```

Comment:
I created a new package so as to not pollute any changes you might have. I created it using vue cli: https://cli.vuejs.org/

I added a script to `copy:assets` ie: get the proper labels support in the app.